src/types.ts

```diff
@@ -446,6 +454,8 @@ export interface RevolutPayOptions extends CommonOptions {
   email?: string
   /** Styles of the RevolutPay button */
   buttonStyle?: ButtonStyleOptions
+  /** Prefiiled customer details. Only valid details are prefilled. */
+  customer?: PrefilledCustomerDetails
```

Comment:
why not: 

```ts 
customer?: CustomerDetails & {dateOfBirth: string}
```
with a tsdoc comment on the string format type for the dob? as with other existing customer details properties? 
![CleanShot 2022-11-08 at 10 58 50](https://user-images.githubusercontent.com/10930234/200534475-3a8ae6e7-73b6-4649-a8ce-c7cb3ede000b.png)

Reply:
Or better still add `dateOfBirth` to the existing `CustomerDetails` type - with a comment stating its used only for revpay?

**These will keep our exposed APIs as consistent as possible**

Reply:
`CustomerDetails` has `name`. How would we get `firstName` and `lastName` in this case? I'm not keen to using `.split(' ')`